streampipes-extensions-management/src/main/java/org/apache/streampipes/extensions/management/connect/adapter/format/json/arraykey/JsonParser.java

```diff
@@ -30,11 +30,12 @@
 import org.apache.streampipes.model.schema.EventProperty;
 import org.apache.streampipes.model.schema.EventSchema;
 
-import jakarta.json.Json;
-import jakarta.json.stream.JsonParserFactory;
```

Comment:
With applying checkstyle to the whole code base this should not occur in that way, right?
Have we somewhere a misconfiguration?

Reply:
Thanks for reviewing!
Yes, the imports should be compatible.
I modified the checkstyle config so that there are no changes to the import order.